src/components/buttons/CopyButton/CopyButton.tsx

```diff
@@ -35,6 +41,8 @@ export interface ICopyButtonProps extends ILocalContainerProps {
 	uncopiedStateIconVariant?: UncopiedStateIconVariants | keyof typeof UncopiedStateIconVariants | null;
 	/* Icon to show in copied state */
 	copiedStateIconVariant?: CopiedStateIconVariants | keyof typeof CopiedStateIconVariants | null;
+	/* Background styling for uncopied state */
+	uncopiedStateBGStyleVariant?: UncopiedStateBGStyleVariants | keyof typeof UncopiedStateBGStyleVariants;
```

Comment:
I know you were just following a pattern here, but do you think it's necessary to have `UncopiedStateBGStyleVariants` in addition to the `keyof typeof UncopiedStateBGStyleVariants` ? Seems like just the `keyof typeof UncopiedStateBGStyleVariants` would be enough... unless I'm missing something?

Reply:
Don't think you're missing anything, pretty sure we only want `keyof typeof`, though below a prop is being passed `as CopiedStateIconVariants`, but removing the union doesn't throw an error, so might as well